### DISCUSSION TITLE
Docs: How can we add a parameter in the translate

### DISCUSSION THREAD

**Discussion by k61b** [vercel/next.js#52493] [Help]
### What is the improvement or update you wish to see?

We want an example of localization on localization example. We have integrated app router translate example like [this](https://github.com/vercel/next.js/tree/canary/examples/app-dir-i18n-routing). But we don't know how can we add a parameter in the translate text.

### Is there any context that might help us understand?

Ability to pass parameters to localization texts.

### Does the docs page already exist? Please link to it.

https://github.com/vercel/next.js/tree/canary/examples/app-dir-i18n-routing

**Comment by SJ-Kumar** [comment]
Please check if the following page will be helpful....
https://nextjs.org/docs/pages/building-your-application/routing/internationalization

**Comment by k61b** [comment]
I couldn't see a paragraph about dynamic translate texts here?
